groups/ecs/ecs.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_ecs_cluster" "grafana_cluster" {
+  name = "${var.service}-cluster"
+}
+
+resource "aws_ecs_task_definition" "grafana_task" {
+  family                = "${var.service}-task"
+  network_mode          = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu = "256"
+  memory = "512"
+
+  execution_role_arn = aws_iam_role.ecs_execution_role.arn
+  task_role_arn = aws_iam_role.ecs_task_role.arn
+
+  container_definitions = <<DEFINITION
+  [
+    {
+      "name": "grafana-container",
+      "image": "${local.image_owner_id}.dkr.ecr.${var.region}.amazonaws.com/${var.image_repository_name}:${var.grafana_image_version}",
+      "essential": true,
+      "portMappings": [
+        {
+          "containerPort": 3000,
+          "hostPort": 3000
+        }
+      ]
+    }
+  ]
+  DEFINITION
+}
+
+resource "aws_ecs_service" "grafana_service" {
+  name            = "${var.service}-service"
+  cluster         = aws_ecs_cluster.grafana_cluster.id
+  task_definition = aws_ecs_task_definition.grafana_task.arn
+  launch_type     = "FARGATE"
+  depends_on = [
+    aws_iam_role.ecs_execution_role,
+  ]
+  desired_count = 1
```

Comment:
Ideally this would be a variable defaulted to 1 to allow for overrides if needed.